readme change to help windows users (#372)
codeforpdx/PASS: README.md

   Currently we require Node version 16 or higher and NPM for our package manager.  Most places recommend using a node version manager to install node and npm. To proceed using NVM perform the following.. 

> 1. Download NVM for your system. Find instructions here: https://github.com/nvm-sh/nvm (NOTE: for window users use this link: https://github.com/coreybutler/nvm-windows)

Reviewer: Good find overall. I think we can mainline this comment though. Doesn't need to be a parenthesis

```suggestion
1. Download NVM for your system. 
  - For Mac, Linux, and other POSIX users: https://github.com/nvm-sh/nvm 
  - For Windows users: https://github.com/coreybutler/nvm-windows
```

Reviewer 2: Yeah, I like this approach with the bullet points
